features/apimgt/org.wso2.carbon.apimgt.publisher.feature/src/main/resources/publisher/source/src/app/data/AuthManager.js

```diff
@@ -122,9 +122,15 @@ class AuthManager {
             .then(response => response.json())
             .then((data) => {
                 let user = null;
+                let username;
                 if (data.active) {
                     const currentEnv = Utils.getCurrentEnvironment();
-                    user = new User(currentEnv.label, data.username);
+                    if (data.username.includes('@carbon.super')) {
```

Comment:
is there an "endswith()" function?

Reply:
Yes. I have replaced the includes method with endsWith()